### PR TITLE
Fix site uri blue-oil.org to blueoil.org

### PR DESCRIPTION
**Pull request by suttang**
## Motivation and Context

We got new domain `blueoil.org`, so change site URI to new one.

## Caution

Please do not merge until configuration of the following domains is complete.

- https://jenkins.blueoil.org
- https://docs.blueoil.org

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 ---
 
-[![Build Status](https://jenkins.blue-oil.org/job/blueoil_main/badge/icon)](https://jenkins.blue-oil.org/job/blueoil_main/)
-[![Build Status](https://jenkins.blue-oil.org/job/blueoil_lmnet/badge/icon)](https://jenkins.blue-oil.org/job/blueoil_lmnet/)
+[![Build Status](https://jenkins.blueoil.org/job/blueoil_main/badge/icon)](https://jenkins.blueoil.org/job/blueoil_main/)
```
Comment (iizukak) [inline README.md]: Sorry. CI badge is avairable ?

Comment (suttang) [reply README.md]: No. in now, jenkins.blueoil.org is unavailable.
before merge this PR, we need to setup https://jenkins.blueoil.org and https://docs.blueoil.org first.

**Comment by iizukak** [comment]
@suttang I see.

**Comment by suttang** [comment]
@iizukak Now, Jenkins icon is available!! https://jenkins.blueoil.org/job/blueoil_main/badge/icon

**Comment by iizukak** [comment]
@suttang thanks.

**Pull request merged**